caproto/ioc_examples/mocking_records.py

```diff
@@ -3,26 +3,38 @@
 
 
 class RecordMockingIOC(PVGroup):
-    # Define two records, an analog input (ai) record:
+    # Define three records, an analog input (ai) record:
     A = pvproperty(value=1.0, mock_record='ai')
     # And an analog output (ao) record:
     B = pvproperty(value=2.0, mock_record='ao',
                    precision=3)
+    # and an ai with all the bells and whistles
+    C = pvproperty(value=0.0,
+                   mock_record='ai',
+                   upper_alarm_limit=2.0,
+                   lower_alarm_limit=-2.0,
+                   upper_warning_limit=1.0,
+                   lower_warning_limit=-1.0,
+                   upper_ctrl_limit=3.0,
+                   lower_ctrl_limit=-3.0,
+                   units="mm",
+                   precision=3)
 
     @B.putter
     async def B(self, instance, value):
         if value == 1:
-            # Mocked record will pick up the alarm status simply by use raising
-            # an exception in the putter:
+            # Mocked record will pick up the alarm status simply by
+            # use raising an exception in the putter:
             raise ValueError('Invalid value!')
 
-    # It's also possible to modify some of the behavior of fields on a per-
-    # record basis. The following function is called whenever A.RVAL is put to:
+    # It's also possible to modify some of the behavior of fields on a
```

Comment:
At the risk of being nitpicky, can we forego submitting formatting changes when the original source is already PEP8-compliant?

```
In [2]: len("    # It's also possible to modify some of the behavior of fields on a per-")
Out[2]: 75
In [3]: len("    # It's also possible to modify some of the behavior of fields on a")
Out[3]: 70
```

Reply:
I was just trusting emacs paragraph re-flow

Reply:
Does it touch lines that you don't otherwise edit?

Reply:
Resolved (for now) after the most recent rebase. Arbitrary sub-79 char wrapping discussion will continue in other channels...